rt/bat/RUNTESTOPT.BAT

```diff
@@ -0,0 +1,30 @@
+@echo off
+setlocal
+pushd %~dp0
+cd..
+cd..
+
+rem
+rem this one should assemble okay
+call bat\asm   rt\mlc\testopt @rt\opt\testopt0.opt
+
+rem
+rem this one should assemble with error for invalid option
+call bat\asm   rt\mlc\testopt @rt\opt\testopt1.opt
+
+rem
+rem this one should assemble okay with valid reference
+call bat\asm   rt\mlc\testopt @rt\opt\testopt2.opt
+
+rem
+rem this one should assemble with error for incorrect reference
+call bat\asm   rt\mlc\testopt @rt\opt\testopt3.opt
+
+rem
+rem this one should assemble with error for self-reference
+call bat\asm   rt\mlc\testopt @rt\opt\testopt4.opt
+
+rem
+rem this one should assemble with error for circular reference
+call bat\asm   rt\mlc\testopt @rt\opt\testopt5.opt
+
```

Comment:
While the bat file does run to completion, it requires user interaction to determine whether it runs successfully, so it cannot be added to the BUILD.BAT tests. Anthony created more robust bat files that are invoked in BUILD.BAT. See bat/RUNASSIST.BAT for one example.